views/apis.jade

```diff
@@ -39,7 +39,7 @@ mixin apiDef(anApi)
                 img.pull-right(src='/images/key-icon-32.png' title='API is secured over API keys')
 
       .box-body
-        a(href='/apis/#{anApi.id}')
+        a(href='/apis/#{anApi.id}' title="#{anApi.name}")
```

Comment:
Follow the existing standard and use single quote please

Reply:
If you look at some of the files there is no existing standard of single quotes some files have single quotes some have double quotes, and this is on the initial commit of the files with no changes from anyone